styles.css

```diff
@@ -423,3 +388,280 @@ hr {
+  }
+
+  .action_btn_wrapper {
+    display: flex;
+    width: 100%;
+    justify-content: center;
+    align-items: center;
+    position: absolute;
+    bottom: 8%;
+    left: 0;
+    right: 0;
+  }
+
+  .scroll_button {
+    width: 42px;
+    height: 42px;
+    border-radius: 50%;
+    border: none;
+  }
+
+  .scroll_button:hover {
+    background-color: green;
+    box-shadow: 0 8px 16px rgba(54, 179, 127, 0.24);
+  }
+}
+
+/* dekstop */
+@media screen and (min-width: 768px) {
+  /* header section */
+  .header {
+    height: 72px;
+    background-color: transparent;
+  }
+
+  .header img {
+    display: none;
+  }
+
+  .header_container {
+    display: flex;
+    justify-content: center;
+    width: 90%;
+    align-items: center;
+  }
+
+  .header nav {
+    width: 100%;
+    display: flex;
+    justify-content: center;
+  }
+
+  .header nav ul {
+    padding: 0;
+    width: 50%;
+    display: flex;
+    justify-content: center;
+    gap: 8px;
+    list-style: none;
+  }
+
+  .header nav ul li a {
+    padding: 10px 12px;
+    text-decoration: none;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+  }
+
+  .header::before {
+    content: "";
+    position: absolute;
+    width: 100%;
+    height: 140vh;
+    z-index: -1;
+    background-image: url(assets/bg_desktop.png);
+    background-position: top;
+    background-size: cover;
+    background-repeat: no-repeat;
+  }
+
+  /* headline section */
+  .headline {
+    display: flex;
+    justify-content: center;
+    background-image: none;
+    padding-top: 10vh;
+  }
+
+  .headline_container {
+    width: 770px;
+  }
+
+  .desktop_hidden {
+    display: none;
+  }
+
+  .action_btn_wrapper {
+    bottom: 30%;
+  }
+
+  /* work section */
+
+  .work_card_container {
+    margin: auto;
+    display: grid;
+    width: 60%;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: 1fr 1fr;
+    row-gap: 24px;
+    column-gap: 32px;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .project_card {
+    height: 474px;
+    width: 369px;
+    background-color: #fff;
+    border: 1px solid #d0d9d4;
+    border-radius: 8px;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .h3_my_work {
+    font-family: "Inter", sans-serif;
+    text-align: center;
+    font-size: 40px;
+    color: #172b4d;
+    font-weight: 700;
+    margin-top: 20%;
+    margin-bottom: 52px;
+  }
+
+  /* aboutme section */
+
+  .aboutme {
+    margin-top: 8%;
+    width: 770px;
+  }
+
+  .aboutme::before {
+    position: absolute;
+    content: "";
+    background: url(assets/bg2_about_desktop.svg);
+    width: 224px;
+    height: 245px;
+    top: -25%;
+    right: -72%;
+  }
+
+  .aboutme::after {
+    position: absolute;
+    content: "";
+    background: url(assets/bg1_about_desktop.svg);
+    width: 236px;
+    height: 128px;
+    bottom: -0%;
+    left: -72%;
+    z-index: -1;
+  }
+
+  .aboutme_box_container {
+    flex-direction: row;
+    justify-content: center;
+  }
```

Comment:
Consider working on the layout of this section to make it adaptable to the chosen media breakpoint. Currently all your cards break into one column and in some larger breakpoints two column which is not in accordance with the design guidelines. I suggest fixing this in order to make you design similar to the original template which is always three columns beginning from the desktop and one column on mobile.

one column 
<img width="573" alt="hmmm" src="https://github.com/Arslan49sp/PortfolioMicro/assets/84930355/e85f7f69-5714-43d8-906f-7596e973fdc9">

Two column
<img width="795" alt="ds" src="https://github.com/Arslan49sp/PortfolioMicro/assets/84930355/26e747d7-ead5-4b6b-a8aa-a22dbbf42feb">

HINT: I believe the width on the cards is still the cause of this problem. You should always use 100% for thing you'd want to  be able to resize as the screen changes